SU-SWS/stanford_profile
# D8CORE-1933: removed bottom margin from last intro style

# READY FOR REVIEW

# Summary
- When the intro text is last in the paragraph there's a bunch of white space until the next paragraph. This removes the bottom margin on the intro text style.

![Screen Shot 2020-05-06 at 5 47 19 PM](https://user-images.githubusercontent.com/284440/81242334-b4066700-8fc1-11ea-93b8-21cb7cec1d9a.png)


# Needed By (Date)
Soon.

# Urgency
- Not urgent

# Steps to Test

1. Check out this branch
1. Create a first text paragraph ending with intro text style
1. Create a subsequent text paragraph
1. Verify that the margin on the first text paragraph has no bottom margin
1. Create a paragraph with intro text style followed by other  text.
1. Verify that there is bottom margin on the intro text style.

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1933
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
